src/templates/blog-profile.js

```diff
@@ -64,7 +64,7 @@ export const pageQuery = graphql`
     }
     allMarkdownRemark(
       filter: {
-        frontmatter: {author: {id: {eq:  $slug}}},
+        frontmatter: {authors: {elemMatch: {id: {eq:  $slug}}}},
```

Comment:
elemMatch? I've not heard of that yet - I'll have to check it out :)

Reply:
is doesn't seem to be a thing, but I think elemMatch is what you're looking for.